swap/src/monero.rs

```diff
@@ -360,6 +370,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn create_monero_decimal() {
+        let monero = "10";
+        let amount = Amount::parse_monero(monero).unwrap();
+        let monero_decimal = amount.as_monero_decimal();
+        assert_eq!(Decimal::new(10, 0), monero_decimal);
+    }
+
+    #[test]
+    fn create_piconero_offset_decimal() {
+        let monero = "10";
+        let amount = Amount::parse_monero(monero).unwrap();
+        let monero_decimal = amount.as_monero_decimal();
+        assert_eq!(Decimal::new(10, 0), monero_decimal);
+    }
```

Comment:
The same test twice with a different name?

Reply:
yeah sorry I actually made an erroneous commit